pages/coinbase/homepage.py

```diff
@@ -30,27 +40,242 @@ def __init__(self, driver, spawn=True):
+    status_link_loc =               (by.XPATH, "//a[.='Status']")
+    taxes_link_loc =                (by.XPATH, "//a[.='Taxes']")
+
+    # COMPANY FOOTER
+
+    about_link_loc =                (by.XPATH, "//a[.='About']")
+    affiliates_link_loc =           (by.XPATH, "//a[.='Affiliates']")
+    careers_link_loc =              (by.XPATH, "//a[.='Careers']")
+    partners_link_loc =             (by.XPATH, "//a[.='Partners']")
+    press_link_loc =                (by.XPATH, "//a[.='Press']")
+    legal_and_privacy_link_loc =    (by.XPATH, "//a[.='Legal & Privacy']")
+    cookie_policy_link_loc =        (by.XPATH, "//a[.='Cookie Policy']")
+    support_link_loc =              (by.XPATH, "//a[.='Support']")
+
+    # SOCIAL FOOTER
+
+    blog_link_loc =                 (by.XPATH, "//a[.='Blog']")
+    twitter_link_loc =              (by.XPATH, "//a[.='Twitter']")
+    facebook_link_loc =             (by.XPATH, "//a[.='Facebook']")
+
 
     def build_elements(self):
 
+        # COOKIE BANNER
+
+        self.cookie_banner_dismiss_button =  Button(self.driver, self.cookie_banner_dismiss_button_loc)
+
         # NAVIGATION BAR
 
-        self.home_link = Link(
-            self.driver, self.home_link_loc, destination='home')
-        self.prices_link = Link(self.driver, self.prices_link_loc)
-        self.products_menu = ReactiveMenu(self.driver,self.products_menu_loc)
-        self.company_menu = ReactiveMenu(self.driver, self.company_menu_loc)
-        self.earn_crypto_link = Link(self.driver, self.earn_crypto_link_loc)
-        self.sign_in_link = Link(
-            self.driver, self.sign_in_link_loc, destination='signin')
-        self.header_get_started_button = Button(
-            self.driver, self.header_get_started_button_loc)
+        self.home_link =       Link(self.driver, self.home_link_loc)
+        self.prices_link =     Link(self.driver, self.prices_link_loc)
+        self.products_menu =   ReactiveMenu(self.driver,self.products_menu_loc)
+        self.company_menu =    ReactiveMenu(self.driver, self.company_menu_loc)
+        self.earn_crypto_link= Link(self.driver, self.earn_crypto_link_loc)
+        self.sign_in_link =    Link(self.driver, self.sign_in_link_loc)
+        self.header_get_started_button = Button(self.driver,
+                                            self.header_get_started_button_loc)
+
+        # CREATE ACCOUNT
+
+        self.email_address_field = Field(self.driver,
+                                         self.email_address_field_loc)
+        self.get_started_button =  Button(self.driver,
+                                          self.get_started_button_loc)
+
+        # CRYPTO CURRENCY MENU
+  
+        self.bitcoin_text_link =  Link(self.driver,self.bitcoin_text_link_loc)
+        self.bitcoin_BTC_link =   Link(self.driver,self.bitcoin_BTC_link_loc)
+        self.bitcoin_logo_link =  Link(self.driver,self.bitcoin_logo_link_loc)
+        self.bitcoin_buy_button = Button(
```

Comment:
Your indentation is very inconsistent.

I recommend doing something that almost always works. I do this:

```
self.bitcoin_buy_button = Button(
    self.driver,self.bitcoin_buy_button_loc)
```

First line ends at open parenthesis, second line begins a single tab after the first. It nearly always fits, so I can do the same thing every time and it looks neater. You're the main author of this file so you don't have to do what I did, but you should pick something versatile and stick with it.